src/citrine/resources/data_objects.py

```diff
@@ -31,6 +32,8 @@ class DataObject(DataConcepts, ABC):
 class DataObjectCollection(DataConceptsCollection[DataObjectResourceType], ABC):
     """A collection of one kind of data object object."""
 
+    @deprecated(deprecated_in="0.130.0", removed_in="1.0.0",
```

Comment:
oh noes, why are we adding deprecation already?

Reply:
This method should have been deprecated all along, but somebody neglected to mark it as such. Hence we need to put it through a property deprecation cycle.